datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
Do you mean this spot?
https://github.com/datalad/datalad/blob/master/datalad/crawler/nodes/annex.py#L148